extensions/machine-learning/src/prediction/predictService.ts

```diff
@@ -173,7 +173,7 @@ WITH predict_input
 AS (
 	SELECT TOP 1000
 	${this.getInputColumnNames(columns, 'pi')}
-	FROM [${utils.doubleEscapeSingleBrackets(sourceTable.databaseName)}].[${sourceTable.schema}].[${utils.doubleEscapeSingleBrackets(sourceTable.tableName)}] as pi
+	FROM [${utils.doubleEscapeSingleBrackets(sourceTable.databaseName)}].[${sourceTable.schema}].[${utils.doubleEscapeSingleBrackets(sourceTable.tableName)}] AS pi
```

Comment:
I keep thinking is there no way to take out these queries and have them in sperate const or so to take it away from the service code...?

Reply:
not a PR blocker but for later...might be easy to understand and update the code...

Reply:
sure I can move them to a different module